src/api/transfers/routes.js

```diff
@@ -72,5 +73,46 @@ module.exports = [{
       }
     }
   }
+},
+{
+  method: 'PUT',
+  path: '/transfers/{id}',
+  handler: Handler.fulfillTransfer,
```

Comment:
please follow the British standard for the fulfil, fulfilment

find and replace may help, please do it for case sensitive replaces